src/app/functions/universal/dataBroker.ts

```diff
@@ -0,0 +1,17 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+export type DataBrokerRemovalStatus =
+  | "active"
+  | "on_hold"
+  | "ceased_operation"
+  | "scan_under_maintenance"
+  | "removal_under_maintenance";
```

Comment:
Nit: `DataBrokerRemovalStatus` could also be derived from `DataBrokerRemovalStatusMap`:

```suggestion
export type DataBrokerRemovalStatus = (typeof DataBrokerRemovalStatusMap)[keyof typeof DataBrokerRemovalStatusMap];
```